lib/js/jsconsole.nim

```diff
@@ -0,0 +1,32 @@
+#
+#
+#            Nim's Runtime Library
+#        (c) Copyright 2012 Andreas Rumpf
+#
+#    See the file "copying.txt", included in this
+#    distribution, for details about the copyright.
+#
+
+## Wrapper for the `console` object for the `JavaScript backend
+## <backends.html#the-javascript-target>`_.
+
+when not defined(js) and not defined(Nimdoc):
+  {.error: "This module only works on the JavaScript platform".}
+
+type Console* {.importc.} = ref object of RootObj
+
+{.push importcpp .}
+
+proc log*[A](console: Console, a: A)
```

Comment:
@andreaferretti, I've tested `console.log.apply(null, [["hi"], ["bye", "yo"]])` in Firefox and it worked perfectly fine for me. Could you please elaborate on whats wrong with your Firefox? =) I'm not really happy that arguments are getting wrapped to an array on their way to `console.log`, like you did in your version, while my version does "unpack" the arguments before calling the function.
As a side note it would be nice to have some way of passing converted varargs to js function without wrapping them into an array. What do you think, @Araq?